_drafts/2017-10-13-rprofile-david-smith.md

```diff
@@ -0,0 +1,119 @@
+---
+name: rprofile-david-smith
+layout: post_discourse
+title: ".rprofile: David Smith"
+date: 2017-10-13
+authors:
+  - name: Kelly O'Briant
+    url: https://kellobri.github.io/
+  - name: Stefanie Butland
```

Comment:
Please remove my name as author. Thank you for suggesting it but this is all you Kelly!!